app/views/static_pages/about/_overview.html.erb

```diff
@@ -1,51 +1,54 @@
-<section class="overview odin-dark-bg">
+<section class="py-12 sm:py-24 bg-gray-100">
```

Comment:
[BUG] We'll need `odin-dark-bg` on this to override the gray on dark mode.

```suggestion
<section class="py-12 sm:py-24 bg-gray-100 odin-dark-bg">
```

<img width="1916" alt="Screenshot 2022-07-18 at 01 02 14" src="https://user-images.githubusercontent.com/7963776/179430180-96be6730-3331-4c8e-ba0b-15c8681ffab4.png">